tests/unit/backend/utils/field-validation/email-validation.spec.ts

```diff
@@ -3,6 +3,8 @@ import EmailValidator from 'src/app/utils/field-validation/validators/EmailValid
 import { BasicField } from 'src/types/field/fieldTypes'
 import { ISingleAnswerResponse } from 'src/types/response'
 
+import { ValidateFieldError } from '../../../../../dist/backend/app/modules/submission/submission.errors'
```

Comment:
:( missed this buggy import here when I reviewed, only discovered when I tried to split out the tests in #742 haha. please dont import from `dist` in TypeScript tests!

Reply:
ok got it, thanks for letting me know